Source/WebCore/html/HTMLBodyElement.cpp

```diff
@@ -3,7 +3,8 @@
  *           (C) 1999 Antti Koivisto (koivisto@kde.org)
  *           (C) 2000 Simon Hausmann (hausmann@kde.org)
  *           (C) 2001 Dirk Mueller (mueller@kde.org)
- * Copyright (C) 2004-2021 Apple Inc. All rights reserved.
+ * Copyright (C) 2004-2023 Apple Inc. All rights reserved.
+ * Copyright (C) 2014 Google Inc. All rights reserved.
```

Comment:
This is a code removal, so not sure there's any Google IP involved here.

Reply:
I was just merging Google patch and using it as reference, hence I added it. If you want, I can get rid of it.

Reply:
Technically, you are right, it is not merge but just removal and we are just using it as inspiration to follow. Noted, I am pushing updated commit to remove this line and also improve commit message to change "Merge" to "Inspired".